packages/location-management/src/components/LocationUnitView/tests/__snapshots__/table.test.tsx.snap

```diff
@@ -0,0 +1,53 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Location-module/locationunit renders without crashing 1`] = `
+Object {
+  "children": <Table
+    data={
+      Array [
+        Object {
+          "geographicLevel": 1,
+          "id": "1",
+          "key": "1",
+          "name": "Edrward 1",
+        },
+        Object {
+          "geographicLevel": 2,
+          "id": "2",
+          "key": "2",
+          "name": "Edrward 2",
+        },
+        Object {
+          "geographicLevel": 3,
+          "id": "3",
+          "key": "3",
+          "name": "Edrward 3",
+        },
+        Object {
+          "geographicLevel": 4,
+          "id": "4",
+          "key": "4",
+          "name": "Edrward 4",
+        },
+      ]
+    }
+  />,
+  "history": Object {
```

Comment:
Can we do `.find('Table')` and avoid including history in the snapshot?

Reply:
i that way we might miss checking the children let me optimize this snapshot

Reply:
done